src/shared/installationVerification.ts

```diff
@@ -54,6 +54,7 @@ export class CodeVerifierInfo {
     this.data = value;
   }
 
+  // eslint-disable-next-line @typescript-eslint/member-ordering
```

Comment:
See: https://github.com/typescript-eslint/typescript-eslint/issues/4576

updating the lint rules to alphabetically sort getters and setters fix the error but I'm not sure if that change would cause more updates to the ongoing changes in our repos.